documentation-website/Writerside/topics/Data-Types.topic

```diff
@@ -290,6 +290,50 @@
                 it[budgets] = listOf(9999.0)
             }
         </code-block>
+        <chapter title="Multi-dimensional arrays" id="how-to-use-multi-dimensional-array-types">
+            <p>PostgreSQL database supports the explicit ARRAY data type, which includes support for multi-dimensional arrays.</p>
+            <p>Exposed supports columns defined as multi-dimensional arrays, with the stored contents being any
+                out-of-the-box or custom data type.
```

Comment:
Could you please also adjust the starting lines in the section before (Exposed currently supports only one-dimension...), as it counters what is said here.